src/lib.rs

```diff
@@ -45,10 +45,25 @@ pub trait Geometry {
 }
 
 /// Default (recommended) Angular Threshold is 180 degrees, which means threshold disabled.
+#[cfg(feature = "experimental")]
 pub fn generate_tangents_default<I: Geometry>(geometry: &mut I) -> bool {
     unsafe { generated::genTangSpace(geometry, 180.0) }
 }
 
+#[cfg(not(feature = "experimental"))]
+/// Default (recommended) Angular Threshold is 180 degrees, which means threshold disabled.
+pub fn generate_tangents_default<I: Geometry>(_: &mut I) -> bool {
+    panic!(
```

Comment:
This isn't what I had in mind. I expected the experimental feature to build and use the generated Rust code (and subsequently neither build nor run the C code) when enabled.

Reply:
Think of the feature as choosing the 'backend' for tangent generation.

Reply:
ah, makes sense, haha.